source/includes/_endpoint-dataset.md

```diff
@@ -565,3 +565,23 @@ PATCH the "expression" attribute to modify. An empty "expression" object, like
 ##### Decks
 
 ##### Permissions
+
+##### Primary key
+
+`/datasets/{id}/pk`
+
+Setting a primary key on a dataset causes updates (particularly streamed
+updates) mentioning existing rows to be updated instead of new rows being
+inserted.
+
+GET on this resource returns a Shoji Entity with a "pk"" attribute in its body,
```

Comment:
Can you please add examples in json and python for these verbs?.

Also fix the double "" by pk in this line.